docs/content/getting-started/setup-openfga/docker-setup.mdx

```diff
@@ -476,11 +476,11 @@ docker run -p 8080:8080 -p 8081:8081 openfga/openfga -e OPENFGA_EXPERIMENTALS="f
 
 The following table enumerates the experimental flags, a description of what they do, and the versions of OpenFGA the flag is supported in:
 
-| Name              | Description                                                        | OpenFGA Version       | Status                     |
-|-------------------|--------------------------------------------------------------------|-----------------------|----------------------------|
-| otel-metrics      | Enables support for exposing OpenFGA metrics through OpenTelemetry | `0.3.2 <= v < v0.3.5` | became default in `v0.3.5` |
-| list-objects      | Enables ListObjects API                                            | `0.2.0 <= v < v0.3.3` | became default in `v0.3.3` |
-| check-query-cache | Enables caching of check subproblem result                         | `1.3.1 <= v `         | currently experimental     |
+| Name              | Description                                                        | OpenFGA Version       | Status                        |
+|-------------------|--------------------------------------------------------------------|-----------------------|-------------------------------|
+| otel-metrics      | Enables support for exposing OpenFGA metrics through OpenTelemetry | `0.3.2 <= v < v0.3.5` | became default in `v0.3.5`    |
+| list-objects      | Enables ListObjects API                                            | `0.2.0 <= v < v0.3.3` | became default in `v0.3.3`    |
+| check-query-cache | Enables caching of check subproblem result                         | `1.3.1 <= v < v1.3.6` | became supported from `v1.3.6`|
```

Comment:
Here and above, I'm not sure what this is supposed to mean. I wonder if we can drop the "Status" column in favor of just the explicit version range?

An experimental flag is present in a release until we remove it. Once we remove it I wouldn't say it is "defaulted", because it is is gone. It no longer is interpreted in any way by the server. So I think it is less confusing if we just say the range of OpenFGA server versions that the experimental flag existed in.

WDYT?

Reply:
Agree - I have trouble coming up with what to write in the status section.  What do you think @rhamzeh ?